src/cli/Repl.php

```diff
@@ -300,6 +300,11 @@ private function intercept($command)
         if (isset($variable[1])) {
             return $this->handleShowType($variable[1]);
         }
+
+        $this->console->setColor(Console::FG_RED);
+        $this->console->writeln("I don't know the command `$command'. Sorry!");
+        $this->console->writeln("Available commands: :clear, :quit, :what and :t `symbol`");
```

Comment:
Quack compiler uses i18n to provide messages in the user language. So on, we could [create two messages here](https://github.com/quack/quack/blob/master/src/intl/locales/en_US.json#L2):

``"QUA010": "I don't know the command `%s'. Sorry!"``
``"QUA020": "Available commands: :clear, :quit, :what and :t `symbol'"``

and in the Repl:

```diff
+ use \QuackCompiler\Intl\Localization;
...
+ $this->console->writeln(Localization::message('QUA010', [$command]));
```